imgur-screenshot

```diff
@@ -237,6 +244,18 @@ is_mac() {
   [ "$(uname -s)" = "Darwin" ]
 }
 
+function is_wayland() {
+  if [[ -z $WAYLAND_DISPLAY ]]; then
+      return 1
+  else
+      return 0
+  fi
+}
```

Comment:
Please follow the function notation used in the rest of the file. Also, the `if` and `return` statements are not needed, so it could be shortened like this:

```bash
is_wayland() {
  [ -z "${WAYLAND_DISPLAY}" ]
}
```